.env.example

```diff
@@ -8,3 +8,6 @@ PGADMIN_DEFAULT_EMAIL
 PGADMIN_DEFAULT_PASSWORD
 
 NEST_PORT
+
+JWT_SECRET
+JWT_EXPIRE
```

Comment:
Would appreciate a short comment to be placed above both of these values: what's the expected value for this field? Can you give an example? Is there a docs link available for best practices?